src/lightning_lite/utilities/types.py

```diff
@@ -60,8 +60,6 @@ def rank(self) -> int:
         ...
 
 
-# Inferred from `torch.optim.lr_scheduler.pyi`
-# Missing attributes were added to improve typing
 @runtime_checkable
 class _LRScheduler(_Stateful[str], Protocol):
```

Comment:
We can remove the underscore from this one

```suggestion
class LRScheduler(_Stateful[str], Protocol):
```